cvat/apps/iam/migrations/0001_remove_business_group.py

```diff
@@ -0,0 +1,34 @@
+# Generated by Django 4.2.16 on 2024-10-30 12:03
+from django.conf import settings
+from django.db import migrations
+
+
+BUSINESS_GROUP_NAME = "business"
+USER_GROUP_NAME = "user"
+
+
+def delete_business_group(apps, schema_editor):
+    Group = apps.get_model('auth', 'Group')
+    User = apps.get_model('auth', 'User')
+
+    if Group.objects.filter(name=USER_GROUP_NAME).exists():
+        user_group = Group.objects.get(name=USER_GROUP_NAME)
+        for user in User.objects.all():
+            if user.groups.filter(name=BUSINESS_GROUP_NAME).exists():
+                user_group.user_set.add(user)
```

Comment:
It's inefficient to fetch all users from the database, and also to add the groups one by one. I think something like this would work:

```python
user_group.user_set.add(*User.objects.filter(groups__name="business"))
```